spec/models/ancestor_ref_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe AncestorRef do
+  let(:ancestor_ref) { described_class.new(service_name: "web", ref: "production") }
+  subject { ancestor_ref }
```

Comment:
It doesn't look like `ancestor_ref` the variable is used outside of this one line, so you could inline it

Reply:
done 👍